osbrain/agent.py

```diff
@@ -196,6 +202,10 @@ class Agent():
     handler : dict
         A dictionary in which the key is the socket and the values are the
         handlers for each socket.
+    context : zmq.Context
+        Used internally to create ZMQ socket objects.
+    poller : zmq.Poller
+        Used internally to wait for incoming data from sockets.
```

Comment:
These will be made private soon, so maybe no need to add the "Used internally". In example:

- ZMQ context to create ZMQ socket objects.
- ZMQ poller to wait for incoming data from sockets.

Reply:
Yeah, I planned to change those  descriptions when the attributes are made private.